16/Dockerfile

```diff
@@ -0,0 +1,77 @@
+FROM metabrainz/consul-template-base:v0.19.4
```

Comment:
We use 0.18.5 as consul base in python brainz so might want to check this with zas.

Reply:
Oh! I see this is our own image. Nevermind then.

Reply:
For posterity: I looked to see if there is a more recent stable image with a proper tag (not `latest`), but didn't see any, so left it as before.